__tests__/verify-java.ps1

```diff
@@ -1,11 +1,31 @@
 if (!$args.Count -or !$args[0])
 {
-  throw "Must supply java version argument"
+  throw "::error::Must supply java version argument"
```

Comment:
Looks good! Thanks for submitting this PR.

The only question I have is regarding this `::error::` syntax. Is there any benefit to doing this? I haven't seen any of our other first party actions use this (I want to maintain some level of consistency). Online I can't seen to find anything either.

Reply:
This is https://help.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-error-message, as seen in `@actions/core`'s `error()` too as an implementation detail.

Not sure about the `throw` in PowerShell though, but according to https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_throw?view=powershell-7 this should work too.

Reply:
I played around this, it makes the error message show up red in the logs so it's a pretty nice addition 😃 

![image](https://user-images.githubusercontent.com/16109154/82949790-7e431580-9fa4-11ea-9dbc-e243fcb3ed42.png)

It doesn't fail the step though so `exit 1` is still needed (for bash).

If Powershell it's a redundant as throw will make the logs show up red and it will fail the step so the extra `::errror::` is not needed

![image](https://user-images.githubusercontent.com/16109154/82950252-44beda00-9fa5-11ea-89b4-36215ff7448a.png)